jetty-util/src/main/java/org/eclipse/jetty/util/log/Log.java

```diff
@@ -339,4 +340,18 @@ public static Properties getProperties()
     {
         return __props;
     }
+
+    /** Format a timestamp.
+     * Format a timestamp using the same efficient formatter as used by Jetty implemented loggers
+     * @param time The time to format
+     * @param units The units of the time
+     * @return The formatted time string.
+     */
+    public static String timestamp(long time, TimeUnit units)
```

Comment:
Nope. Not needed, wrong place, implementation detail exposed into a public class.